frontend/src/lib/k8s/apiProxy/apiTypes.ts

```diff
@@ -0,0 +1,309 @@
+import { OpPatch } from 'json-patch';
+import { KubeMetadata, KubeObjectInterface } from '../cluster';
```

Comment:
This brings a circular import with cluster. I wonder if we can remove in this refactor?

Maybe a cluster.types.ts would allow this?

Reply:
types were moved next to places where they're used, queryParameters moved to a new file